doc/schemas/lightning-enableoffer.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../rpc-schema-draft.json",
+  "type": "object",
+  "additionalProperties": false,
+  "rpc": "disableoffer",
+  "title": "Command for re-enabling an offer",
+  "warning": "experimental-offers only",
+  "description": [
+    "The **enableoffer** RPC command enables an offer, after it has been disabled."
+  ],
+  "request": {
+    "required": [
+      "offer_id"
+    ],
+    "properties": {
+      "offer_id": {
+        "type": "hash",
+        "description": [
+          "The id we use to identify this offer."
+        ]
+      }
+    }
+  },
+  "response": {
+    "required": [
+      "offer_id",
+      "active",
+      "single_use",
+      "bolt12",
+      "used"
+    ],
+    "properties": {
+      "offer_id": {
+        "type": "hash",
+        "description": [
+          "The merkle hash of the offer."
+        ]
+      },
+      "active": {
+        "type": "boolean",
+        "enum": [
+          true
+        ],
+        "description": [
+          "Whether the offer can produce invoices/payments."
+        ]
+      },
+      "single_use": {
+        "type": "boolean",
+        "description": [
+          "Whether the offer is disabled after first successful use."
+        ]
+      },
+      "bolt12": {
+        "type": "string",
+        "description": [
+          "The bolt12 string representing this offer."
+        ]
+      },
+      "used": {
+        "type": "boolean",
+        "description": [
+          "Whether the offer has had an invoice paid / payment made."
+        ]
+      },
+      "label": {
+        "type": "string",
+        "description": [
+          "The label provided when offer was created."
+        ]
+      }
+    },
+    "pre_return_value_notes": [
+      "Note: the returned object is the same format as **listoffers**."
+    ]
+  },
+  "author": [
+    "Rusty Russell <<rusty@rustcorp.com.au>> is mainly responsible."
```

Comment:
Who's the author? @vincenzopalazzo or @rustyrussell

Reply:
Rusty OFC I fetch his code inside mine PR regarding the docs :)

So I forget to add the Co-Developed-by: ... thanks for chatting it